test/unit/io.js

```diff
@@ -939,6 +939,21 @@ describe('Input/output', function () {
       });
   });
 
+  it('Convert SVG to PNG at 14.4DPI', function (done) {
+    sharp(fixtures.inputSvg, { density: 14.4 })
+      .toFormat('png')
+      .toBuffer(function (err, data, info) {
+        if (err) throw err;
+        assert.strictEqual('png', info.format);
+        assert.strictEqual(20, info.width);
+        assert.strictEqual(20, info.height);
+        fixtures.assertSimilar(fixtures.expected('svg14.4.png'), data, function (err) {
+          if (err) throw err;
+          done();
```

Comment:
Not sure if this should also check the metadata of the resulting image. For some reason the density seemed to be `undefined` for this particular image.